sdk/src/network.rs

```diff
@@ -107,6 +266,7 @@ pub enum Network {
     /// Network presets for Calibration (default pre-mainnet).
     Testnet,
     /// Network presets for a local three-node network.
+    #[deprecated]
```

Comment:
can you remind me why this needs to be deprecated? you mentioned this:
> The PR also deprecates Localnet NetworkConfig, because each network will have different config.

but, when i use the localnet, it always has the same values...so i dont quite follow

Reply:
The idea is to use for tests and also localnet different wallet addresses just to avoid some classic pitfalls when something works with "abc" string but crashing with "abd". Different wallet addresses will result in different contract address. But to keep it convenient the create-localnet script will create a network config TOML file that can be used with `recall` CLI by either setting the `-c` flag or setting envvar `RECALL_NETWORK_CONFIG_FILE`.

The next step is to replace `deploy.sh` script in `ipc` by `create-subnet`, that is used for testnet creation. The idea here is to use exactly the same scripts for creation of any subnets (testnet, localnet, etc.).